### PR TITLE
Print running test name

### PR DESCRIPTION
**Pull request by TimAbraldes**
I find myself adding this locally to be able to quickly determine which
test is timing out.

### CODE REVIEW

File: tests/RunTests.java [hunk 1]
```diff
@@ -132,6 +134,8 @@ void runTest(String name) {
         fail += harness.failed();
         knownFail += harness.knownFailed();
         unknownPass += harness.unknownPassed();
+
+        System.out.println("Completed " + name);
```
Comment (marco-c) [inline tests/RunTests.java]: Nit: this is redundant with harness.report()

File: tests/automation.js [hunk 2]
```diff
@@ -265,6 +265,7 @@ casper.test.begin("unit tests", 20 + gfxTests.length, function(test) {
     // Graphics tests
 
     gfxTests.forEach(function(testCase) {
+        console.log("Running " + testCase.name);
```
Comment (marco-c) [inline tests/automation.js]: Nit: Logging for gfx tests shouldn't be needed:
```
[info] [phantom] Step anonymous 91/318 http://localhost:8000/main.html?fontSize=12&midletClassName=gfx/CanvasTest&jars=tests/tests.jar&logConsole=web,page&logLevel=log (HTTP 200)
...
PASS gfx/CanvasTest - Pass
```

**Comment by mykmelez** [comment]
I've wanted something like this too. Just note that it pushes us very close to the 10,000 line limit on Travis, as the latest build on this branch generated 9963 lines of output. And once we breach the limit, Travis will start reporting that tests passed even when they didn't. We should keep an eye on that after landing this, and maybe follow up with a review of the output to see if there's something unuseful that we can pare.

**Comment by brendandahl** [comment]
Getting rid of the verbose output for jar creation would get rid of a good number of lines.

**Pull request merged**